src/acl/external/AD_group/ext_ad_group_acl.cc

```diff
@@ -531,7 +542,7 @@ build_groups_DN_array(const char **array, char *userdomain)
         *entry = My_NameTranslate(wc, source_group_format, ADS_NAME_TYPE_1779);
         safe_free(wc);
         ++array;
-        if (*entry == NULL) {
+        if (!(*entry)) {
```

Comment:
Please do not add parenthesis in such cases:

```suggestion
        if (!*entry) {
```

Reply:
removed. I recall I had added it in response to a compiler error, which has disappeared. Weird, but all the better